### PR TITLE
NH3 CO documentation

### PR DESCRIPTION
**Pull request by roberbike**
Add dfrobot NH3 and CO gravity sensors.
Add initial AQI levels for NH3 and CO.

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -97,7 +97,7 @@ lib_ignore =
 build_flags = 
   ${common.build_flags}
   -D USER_SETUP_LOADED=1
-  -include /$PROJECT_LIBDEPS_DIR/$PIOENV/TFT_eSPI/User_Setups/Setup25_TTGO_T_Display.h
+  -include $PROJECT_LIBDEPS_DIR/$PIOENV/TFT_eSPI/User_Setups/Setup25_TTGO_T_Display.h
```
Comment (hpsaturn) [inline platformio.ini:100]: As long as we have a fix for this PlatformIO issue, please leave this line without changes.

File: lib/gui-utils-tft/src/TFTUtils.hpp [hunk 2]
```diff
@@ -29,13 +29,13 @@
...
 
-enum AQI_COLOR { AQI_NONE, AQI_PM, AQI_CO2 };
+enum AQI_COLOR { AQI_NONE, AQI_PM, AQI_CO2, AQI_CO, AQI_NH3 };
 
 typedef struct GUIData {
     uint8_t mainUnitId;
     uint8_t onSelectionUnit;
-    uint16_t mainValue;
-    uint16_t minorValue;
+    float mainValue;
```
Comment (hpsaturn) [inline lib/gui-utils-tft/src/TFTUtils.hpp:37]: please add a photo of T-Display screen with float numbers, with these new sensors and also with a previous one, like PM2.5.

**Review by hpsaturn — CHANGES_REQUESTED** [review]

File: platformio.ini (hunk 1 above)
Comment (roberbike) [reply platformio.ini]: ok. Done.

File: lib/gui-utils-tft/src/TFTUtils.hpp (hunk 2 above)
Comment (roberbike) [reply lib/gui-utils-tft/src/TFTUtils.hpp:37]: image
image
image
image
image

**Pull request merged**